pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1599,6 +1600,11 @@ def sql(self, sql, table_list=[], algebra=None, return_futures=False, single_gpu
         if (algebra is None):
             algebra = self.explain(sql)
 
+        # when `LogicalValues` appears on the optimized plan there aren't neither BindableTableScan nor TableScan nor Project
+        if "LogicalValues(tuples=[[]])" in algebra:
+            print("This SQL statement returns empty result. Please double check your query.")
+            return
```

Comment:
it should return an empty dataframe this return None which is wrong

Reply:
the algebra at this point does not contain neither BindableTableScan, neither TableScan neither Projects. Then the empty dataframe doesn't have  a column name. Should I use  a  col_name as  'EMPTY' or something like that?

Reply:
cant you just return cudf.DataFrame()

Reply:
import cudf
cudf.DataFrame()

works for me